__mocks__/isomorphic-fetch.js

```diff
@@ -0,0 +1,13 @@
+export const successUrl = '/success'
+export const failureUrl = '/failure'
+
+export default jest.fn(function (url) {
```

Comment:
sweet action! can we test the situation when the promise rejects as well? The Fetch api will reject when there is some sort of network error.